operatorcert/entrypoints/reserve_operator_name.py

```diff
@@ -0,0 +1,96 @@
+import argparse
+import logging
+import sys
+from urllib.parse import urljoin
+
+from operatorcert.pyxis import post_with_cert, get_with_cert
+
+LOGGER = logging.getLogger("operator-cert")
+
+
+def setup_argparser() -> argparse.ArgumentParser:
+    """
+    Setup argument parser
+
+    Returns:
+        Any: Initialized argument parser
+    """
+    parser = argparse.ArgumentParser(
+        description="Reserve the given operator package name"
+    )
+    parser.add_argument("--association", help="Association of the operator package")
+    parser.add_argument("--operator-name", help="Unique name of the operator package")
+    parser.add_argument("--source", help="Source of the operator package")
+    parser.add_argument(
+        "--pyxis-url",
+        default="https://pyxis.engineering.redhat.com/",
+        help="Base URL for Pyxis container metadata API",
+    )
+    parser.add_argument("--cert-path", help="Path to Pyxis auth cert")
+    parser.add_argument("--key-path", help="Path to Pyxis auth key")
+
+    return parser
+
+
+def check_operator_name(args) -> None:
+    rsp = get_with_cert(
+        urljoin(
+            args.pyxis_url,
+            f"v1/operators/packages?filter=package_name=={args.package_name}",
+        ),
+        args.cert_path,
+        args.key_path,
+    )
+
+    if rsp.status_code != 404:
+        package = rsp.json()
```

Comment:
Actually yeah, and because it's an endpoint that returns multiple results, if there's no result it'll just be an empty list, not 404. I'll update it to use `raise_for_status` as well as add handling for a list.